Project.toml

```diff
@@ -1,23 +1,25 @@
 name = "ClimaSeaIce"
 uuid = "6ba0ff68-24e6-4315-936c-2e99227c95a4"
 authors = ["Climate Modeling Alliance and contributors"]
-version = "0.1.4"
+version = "0.2.0"
 
 [deps]
 Adapt = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
+FFMPEG = "c87230d0-a227-11e9-1b43-d7ebe4e7570a"
 KernelAbstractions = "63c18a36-062a-441e-b654-da1e3ab1ce7c"
 Oceananigans = "9e8cae18-63c1-5223-a75c-80ca9d6e9a09"
+Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 RootSolvers = "7181ea78-2dcb-4de3-ab41-2b8ab5a31e74"
 Roots = "f2b01f46-fcfa-551c-844a-d8ac1e96c665"
 SeawaterPolynomials = "d496a93d-167e-4197-9f49-d3af4ff8fe40"
 
 [compat]
 Adapt = "3, 4"
 KernelAbstractions = "0.9"
-Oceananigans = "0.90.11 - 0.99"
-SeawaterPolynomials = "0.3.4"
+Oceananigans = "0.95.3 - 0.99"
```

Comment:
how is this related? maybe you want to put in another PR to tidy things up before this one, which is really major